### PR TITLE
Add knplabs/pagination-bundle recipe

### PR DESCRIPTION
**Pull request by franmomu**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: knplabs/knp-paginator-bundle/2.8/config/packages/knp_paginator.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+knp_paginator:
```
Comment (OskarStark) [inline knplabs/knp-paginator-bundle/2.8/config/packages/knp_paginator.yaml]: You should only provide non default values here

**Comment by Nyholm** [comment]
Ping @franmomu, any updates on Oskar's comment?

**Comment by franmomu** [comment]
oops, I forgot about this, Oskar was right. Taking a look at the docs, maybe I should just add what in the troubleshooting appears:

```
framework:
    translator: { fallbacks: ['%locale%'] }
```

**Comment by Nyholm** [comment]
Any updates on this PR?

#eu-fossa

**Comment by franmomu** [comment]
As you said, it is weird to change another package, so I guess I better close It.

**Pull request closed without merging**